apps/molgenis-components/src/components/layout/MolgenisSession.vue

```diff
@@ -3,20 +3,9 @@
   <div v-else>
     <div>
       <span v-if="session.email && session.email != 'anonymous'">
-        <a
-          href="#"
-          @click.prevent="showChangePasswordForm = true"
-          class="text-light"
-        >
+        <a href="#" @click.prevent="$emit('showAccount')" class="text-light">
```

Comment:
using a button instead of a link for actions avoids the need to stop the nav action , bootstrap has buttons styled as links